core/indigo-core/molecule/smiles_saver.h

```diff
@@ -43,6 +43,11 @@ namespace indigo
     {
     public:
         DECL_ERROR;
+        enum
+        {
+            SMILES_CHEMAXON = 0,
+            SMILES_DAYLIGHT
+        };
```

Comment:
Let's use enum class

Reply:
This is impossible to use class enum in this case. It requires serious architecture refactoring. The enum defined in smiles_saver, but the values required on indigo_options.cpp level, which means we should drag smiles_saver.h, molfiles_saver.h, which has the same kind of enum:

        enum
        {
            MODE_AUTO = 0, // save to v3000 only if the given molecule has any
                           // v3000-specific features
            MODE_2000,     // force saving to v2000 format
            MODE_3000      // force saving to v3000 format
        };